tensorflow/stream_executor/device_description.h

```diff
@@ -154,10 +208,10 @@ class DeviceDescription {
   // Micro Devices, Inc.", or "GenuineIntel".
   const string &device_vendor() const { return device_vendor_; }
 
-  // Returns the CUDA compute capability if we're running on the CUDA platform.
-  // If a CUDA compute capability is not available, the major version will be
-  // zero, and the return value will be false.
-  bool cuda_compute_capability(int *major, int *minor) const;
+  // Returns the hardware version of the device.
```

Comment:
At this point we should just remove the comment.  That's fine, it's obvious what it does.